src/api/fetchAPI.ts

```diff
@@ -0,0 +1,52 @@
+import ERROR_MESSAGE from '../constant/errorMessage';
+
+async function fetchAPI({ url, method, body }: { url: string; method: string; body?: object }) {
```

Comment:
method 타입은 string 보다 더 좁게 잡아주는건 어때용? 물론 ts의 RequestInit 타입내에서는 method 가 string 이긴 하지만 그래도 개발경험 개선을 위해서 한번 시도해봐도 괜찮을 것 같아요ㅎㅎ
 따로 타입으로 분리해서 재활용해도 좋을 것 같고요!
```suggestion
async function fetchAPI({ url, method, body }: { url: string; method: 'GET' | 'POST' | 'PUT' | 'DELETE'  ; body?: object }) {
```

Reply:
오 감사합니다. 2단계 과정에서 `method` 타입을 한 번 잡아보겠습니다 😄